src/datamigration/azext_dms/mgmt/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
Can you put all your sdk code into a `vendored_sdks` folder? i.e. your `mgmt` folder.
This will tell the CI static check to ignore those files for pylint/flake8.